demos/hide/hide.go

```diff
@@ -26,8 +24,8 @@ func (game *GameWorld) Setup(w *ecs.World) {
 	render := engi.NewRenderComponent(texture, engi.Point{8, 8}, "guy")
 	collision := &engi.CollisionComponent{Solid: true, Main: true}
 
-	width := texture.Width() * render.Scale.X
-	height := texture.Height() * render.Scale.Y
+	width := texture.Width() * render.Scale().X
```

Comment:
Also has to do with the OpenGL buffers. Because for most things, the Scale / Drawable doesn't change every frame, we're only re-computing these stuff on-change, and putting it in the GPU RAM (instead of regular RAM), which is way faster.